cmake/functions.cmake

```diff
@@ -50,11 +50,14 @@ function(generate_build_flags)
     if(WOLFSSL_SCTP OR WOLFSSL_USER_SETTINGS)
         set(BUILD_SCTP "yes" PARENT_SCOPE)
     endif()
+    if(WOLFSSL_SRTP OR WOLFSSL_USER_SETTINGS)
+        set(BUILD_SRTP "yes" PARENT_SCOPE)
+    endif()
     set(BUILD_MCAST ${WOLFSSL_MCAST} PARENT_SCOPE)
     set(BUILD_IPV6 ${WOLFSSL_IPV6} PARENT_SCOPE)
     set(BUILD_LEAN_PSK ${WOLFSSL_LEAN_PSK} PARENT_SCOPE)
     set(BUILD_LEAN_TLS ${WOLFSSL_LEAN_TLS} PARENT_SCOPE)
-    set(BUILD_LOWMEM ${WOLFSSL_LOWRESOURCE} PARENT_SCOPE)
+    set(BUILD_LOW_RESOURCE ${WOLFSSL_LOWRESOURCE} PARENT_SCOPE)
```

Comment:
I don't see this used anywhere. Nor is it in configure.ac.

Reply:
BUILD_LOWMEM is set in configure.ac but it doesn't seem to get used there also

Reply:
Agreed. Let's stick with BUILD_LOWMEM for simplicity's sake, since that's what configure.ac has.

Reply:
Still needs to be addressed.